examples/3d/quaternionArcballExample/src/ofApp.cpp

```diff
@@ -63,9 +163,13 @@ void ofApp::mouseDragged(int x, int y, int button){
 	//every time the mouse is dragged, track the change
 	//accumulate the changes inside of curRot through multiplication
 	glm::vec2 mouse(x,y);
-	glm::quat yRot = glm::angleAxis(ofDegToRad(x-lastMouse.x)*dampen, glm::vec3(0,1,0));
-	glm::quat xRot = glm::angleAxis(ofDegToRad(y-lastMouse.y)*dampen, glm::vec3(-1,0,0));
-	curRot = xRot * yRot * curRot;
+	
+	xspeed = ofLerp(xspeed, (x-lastMouse.x) * dampen, 0.1);
+	yspeed = ofLerp(yspeed, (y-lastMouse.y) * dampen, 0.1);
+	
+//	glm::quat yRot = glm::angleAxis(ofDegToRad(x-lastMouse.x)*dampen, glm::vec3(0,1,0));
+//	glm::quat xRot = glm::angleAxis(ofDegToRad(y-lastMouse.y)*dampen, glm::vec3(-1,0,0));
+	//curRot = xRot * yRot * curRot;
     lastMouse = mouse;
```

Comment:
Oops - one more thing. 
Might be good to remove the commented out lines ( 170 - 171 ) and update the comments above?

Reply:
Ok, cleaned up a bit here https://github.com/openframeworks/openFrameworks/pull/7090/commits/a87b958e560cf93a68231f73ae8b1d57448fa04b